### PR TITLE
Logging improvements 

### PR DESCRIPTION
**Pull request by adiochocki**
- Change puts to logger, 
- Add logs on start and end of rake task ,
- Update readme about lokalstack port

Generally app is using logger passed in configuration, for example rails logger: 

```
Eventboss.configure do |config|
  config.logger = Rails.logger
  config.log_level = Rails.configuration.log_level
end
```

So if app will have configured json logs - this library will also log in json format. 

For example logs, running this from cockpit with his logger looks like this: 
Screenshot 2022-03-14 at 09 02 39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,8 +100,8 @@ EVENTBOSS_ENV=env_name # production/staging/test
 EVENTBOSS_REGION=aws_region # i.e. eu-west-1
 EVENTBOSS_CONCURRENCY=10 # default is 25
 
-AWS_SNS_ENDPOINT=http://localhost:4575 # when using with localstack
-AWS_SQS_ENDPOINT=http://localhost:4576 # when using with localstack
+AWS_SNS_ENDPOINT=http://localhost:4566 # when using with localstack
```
Comment (przprz) [inline README.md:103]: 🙇🏻‍♂️

File: lib/eventboss/development_mode.rb [hunk 2]
```diff
@@ -17,6 +17,9 @@ def setup_infrastructure(queues)
           logger.info('development-mode') { "Creating queue #{queue.name}..." }
           sqs_client.create_queue(queue_name: queue.name)
 
+          logger.info('development-mode') { "Creating deadletter queue #{queue.name}-deadletter..." }
```
Comment (przprz) [inline lib/eventboss/development_mode.rb:20]: Do we need these actually?

File: lib/tasks/eventboss.rake [hunk 3]
```diff
@@ -6,6 +6,9 @@ namespace :eventboss do
     task :reload, [:event_name, :source_app, :max_messages] do |task, args|
       source_app = args[:source_app]
       event_name = args[:event_name]
+      start_time = Time.now
```
Comment (przprz) [inline lib/tasks/eventboss.rake:9]: Not sure if we need it - the time will be logged in kibana anyway

**Comment by przprz** [comment]
Looks good, minor comments.
Could you please paste a screenshot that'd show how the log looks like in a sample app?

File: lib/eventboss/development_mode.rb (hunk 2 above)
Comment (adiochocki) [reply lib/eventboss/development_mode.rb:20]: We need this to test for example rake task in development mode. But if its only needed for my testing - I can remove it.

File: lib/tasks/eventboss.rake (hunk 3 above)
Comment (adiochocki) [reply lib/tasks/eventboss.rake:9]: We need this if we want to calculate how long is rake task running. But if we don't need this log - I can remove it. 

But I can remove logging start time and leave calculating total time for task execution.

Comment (adiochocki) [reply lib/tasks/eventboss.rake:9]: Start time log removed, but there is still question do we need log for time execution of rake task.

**Comment by adiochocki** [comment]
> 

Screenshot added.

**Review by nglx — APPROVED** [review]
LGTM 🙇

**Pull request merged**